test/closure.ts

```diff
@@ -0,0 +1,59 @@
+import * as child_process from 'child_process';
```

Comment:
Add a file overview comment explaining what this does, and why we're using it in place of the NPM module?

Reply:
Done.